### ISSUE TITLE
Main IIFE is an arrow function

### ISSUE THREAD

**Issue by nemmtor** [babel/babel-loader#885]
**I'm submitting a bug report**

**Webpack Version:**
5.8.0
**Babel Core Version**:
7.12.9
**Babel Loader Version**:
8.2.2
**Please tell us about your environment:**
Linux

**Current behavior:**
Basically all works good but the whole bundle is wrapped inside IIFE which is an arrow function, I think it shouldn't be like that since im using babel properly - with @babel/preset-env

**Expected/desired behavior:**
Wraps bundle in IIFE normal func

**Comment by vvscode** [comment]
Same for me. Just one point - on webpack `4.44.2` it gives normal function

**Comment by vvscode** [comment]
@nemmtor  it's not loader issue, this function is provided by webpack itself

the link https://webpack.js.org/configuration/output/#outputenvironment was really helpful

```js
  output: {
    path: path.resolve(__dirname, './dist'),
    filename: 'index_bundle.js',

    // next do the trick
    environment: {
      arrowFunction: false, // <-----
    }
  },
```

**Comment by nicolo-ribaudo** [comment]
Yup, that's injected by webpack _after_ running Babel.

I'm keeping this open in case anyone wants to add a mention of this problem and how to solve it to the README.

**Comment by vvscode** [comment]
I will make a PR today

**Comment by JLHwung** [comment]
Fixed in #886
